exporters/logging/src/main/java/io/opentelemetry/exporter/logging/LoggingSpanExporter.java

```diff
@@ -54,7 +54,9 @@ public CompletableResultCode export(Collection<SpanData> spans) {
                   ? ""
                   : instrumentationScopeInfo.getVersion())
           .append("] ")
-          .append(span.getAttributes());
+          .append(span.getAttributes())
+          .append(" ")
+          .append(span.getResource().getAttributes());
```

Comment:
So I'm not completely confident that this is always true, but I do believe that the resource will almost always be the same for all spans in this batch. As a result, the same information will be duplicated in the logs and will get kinda verbose.

For what it's worth, the debug logger in the collector provides output with the resource attributes at the top of the batch, followed by the span content. The output this change provides also doesn't make it super clear which set is attributes vs. resource attributes.